### DISCUSSION TITLE
How to set `ignoreHTTPSErrors` puppeteer?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [ZFC-Digital/puppeteer-real-browser#185] [Q&A] [answered]
I want using self-certified proxy which it will showing alert

![](https://xenit.se/app/uploads/2018-04-08-19_37_51-Chrome_Certificate_warning.png)

usually i use puppeteer like this and working to using my proxy.

```
puppeteer
      .launch({
        headless: true,
        ignoreHTTPSErrors: true,
        slowMo: 0,
        args: [
          "--window-size=1380,800",
          "--remote-debugging-port=9222",
          "--remote-debugging-address=0.0.0.0",
          "--disable-gpu",
          "--disable-features=IsolateOrigins,site-per-process",
          "--blink-settings=imagesEnabled=true"
        ]
      })
```

how to use ignore https certificate validation in this library ?

**Comment by mdervisaygan** [comment, marked as answer]
Can you try this?

```
async function test() {
    const { browser, page } = await connect({
        args: [],
        connectOption: {
            ignoreHTTPSErrors: true,
        }
    })
    await page.goto('<url>')
}

test()
```

**Reply by dimaslanjaka** [reply]
How to change chrome executable path ?

**Reply by mdervisaygan** [reply]
> How to change chrome executable path ?

In order for it to behave like a real browser, the chrome launcher is used to find and use the chrome path on your computer. For this reason, it cannot be changed. When changed, the library is detected when Puppeteer's local path is used. For this reason, a real browser is used.
https://github.com/GoogleChrome/chrome-launcher

**Reply by dimaslanjaka** [reply]
ah i see. thanks then

**Reply by dimaslanjaka** [reply]
@zfcsoftware please refer to my new question https://github.com/zfcsoftware/puppeteer-real-browser/discussions/186#discussion-7261949
